src/components/PeoplePicker/PeoplePicker.tsx

```diff
@@ -1,113 +1,215 @@
-import React from 'react';
-import { IPersonaProps } from '@fluentui/react/lib/Persona';
-import { IBasePickerSuggestionsProps, NormalPeoplePicker } from '@fluentui/react/lib/Pickers';
-import { people } from '@fluentui/example-data';
+import React from "react";
```

Comment:
Same thing as the other file - only import the components being used, rather than the whole library

Reply:
Corrected with above update to change all React imports to selective